include/RAJA/util/Operators.hpp

```diff
@@ -386,7 +386,7 @@ struct logical_and : public detail::comparison_function<Arg1, Arg2>,
   {
     return lhs && rhs;
   }
-  static constexpr const bool identity = true;
+  static constexpr bool identity() {return true;}
```

Comment:
These are pretty standard, they're basically what OpenMP uses IIRC.  I'll look over the list and double check, but the ones I've looked at are good.  The issue with some is that there really isn't an identity value for the operator, which means it can't be used in certain situations without having one specified for the specific problem.